docker-compose/docker-compose.yaml

```diff
@@ -31,6 +31,7 @@ services:
       PROMSCALE_WEB_TELEMETRY_PATH: /metrics-text
       PROMSCALE_DB_URI: postgres://postgres:password@db:5432/postgres?sslmode=allow
       PROMSCALE_TRACING_OTLP_SERVER_ADDRESS: ":9202"
+      PROMSCALE_ENABLE_FEATURE: "tracing"
```

Comment:
This won't be necessary in next promscale release. However since we did not do that release yet, I don't see any reason for not including this.